src/downloader-resolver.ts

```diff
@@ -1,10 +1,15 @@
 import { Downloader } from './Downloader';
 
 export async function getDownloaderForSystem(): Promise<Downloader<any>> {
-  // TODO: Resolve the downloader or default to GotDownloader
-  // Current thoughts are a dot-file traversal for something like
-  // ".electron.downloader" which would be a text file with the name of the
-  // npm module to import() and use as the downloader
-  const { GotDownloader } = await import('./GotDownloader');
-  return new GotDownloader();
+  if (process.env.ELECTRON_GET_TESTSUITE) {
```

Comment:
What's the benefit to this approach rather than mocking `getDownloaderForSystem` in the tests? I'm personally not a fan of injecting test code into the main code, it seems cleaner if `FixtureDownloader` lived in `test/`. With this approach it's not easily discerned from the test code that there's a `FixtureDownloader`.

Reply:
It's mostly because I didn't want to add `proxyquire` to the `devDependencies`.

Reply:
Turns out `proxyquire` is a gigantic pain to deal with when you're using TypeScript and trying to require a TypeScript file. I ended up just setting `downloader` all throughout the `index` spec, which is suboptimal.